DebugUtils/Source/DetourDebugDraw.cpp

```diff
@@ -101,7 +101,8 @@ static void drawPolyBoundaries(duDebugDraw* dd, const dtMeshTile* tile,
 				}
 				for (int m = 0, n = 2; m < 3; n=m++)
 				{
-					if (((t[3] >> (n*2)) & 0x3) == 0) continue;	// Skip inner detail edges.
+					int edgeFlags = (t[3] >> (n * 2)) & 0x3;
```

Comment:
You could make this a helper function too, as it's a little obscure, so you could just call:
```C
if ((rcDetailEdgeFlags(t[3], n) & DT_DETAIL_EDGE_CONTOUR) == 0) continue;
```